ansible/roles/ocp-workload-aiadv-templates/readme.adoc

```diff
@@ -0,0 +1,33 @@
+= ocp-workload-fuse-ignite
```

Comment:
same here

Reply:
@fridim Thanks, addressed using this commit b05b047a4ec79cdccea9a9ee7c0f2eada88d4222